### PR TITLE
Added exec command in sandbox

### PR DESCRIPTION
**Pull request by evalsocket**
# TL;DR
Added exec comamnd in sandbox 

```bash
➜ flytectl sandbox exec -- ls -al
{total 72
drwxr-xr-x   24 root     root           768 Jun 25 17:33 .
drwxr-xr-x    1 root     root          4096 Jun 28 17:37 ..
-rw-r--r--    1 root     root           163 Jun  9 14:28 .flake8
drwxr-xr-x   16 root     root           512 Jun 28 08:18 .git
drwxr-xr-x    4 root     root           128 Jun  9 14:28 .github
-rw-r--r--    1 root     root           234 Jun 22 20:51 .gitignore
-rw-r--r--    1 root     root           318 Jun 25 16:10 .goreleaser.yml
drwxr-xr-x   11 root     root           352 Jun 28 15:15 .idea
-rw-r--r--    1 root     root           482 Jun  9 14:28 .readthedocs.yml
-rw-r--r--    1 root     root           143 Jun  9 14:28 CODEOWNERS
-rw-r--r--    1 root     root           158 Jun  9 14:28 CODE_OF_CONDUCT.md
-rw-r--r--    1 root     root         11341 Jun  9 14:28 LICENSE
-rw-r--r--    1 root     root          2933 Jun 25 16:10 Makefile
-rw-r--r--    1 root     root           184 Jun  9 14:28 NOTICE
-rw-r--r--    1 root     root          4240 Jun 25 16:10 README.md
drwxr-xr-x    5 root     root           160 Jun 22 15:20 boilerplate
drwxr-xr-x   26 root     root           832 Jun 25 16:10 cookbook
-rw-r--r--    1 root     root            44 Jun  9 14:28 katacoda.yaml
drwxr-xr-x    6 root     root           192 Jun  9 14:28 legacy
-rw-r--r--    1 root     root            20 Jun  9 14:28 prquantifier.yaml
-rw-r--r--    1 root     root            93 Jun 25 17:33 registries.yaml
drwxr-xr-x    2 root     root            64 Jun 23 05:55 release-snacks
drwxr-xr-x    3 root     root            96 Jun 12 08:34 tutorial
drwxr-xr-x    6 root     root           192 Jun 21 18:45 venv

```

## Type
 - [ ] Bug Fix
 - [x] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
https://github.com/lyft/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/lyft/flyte/issues/<number>_

### CODE REVIEW

File: cmd/sandbox/exec.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+const (
+	execShort = "Get the status of the sandbox environment."
+	execLong  = `
+Status will retrieve the status of the Sandbox environment. Currently FlyteSandbox runs as a local docker container.
+This will return the docker status for this container
+Usage
+::
+ bin/flytectl sandbox status 
+`
+)
```
Comment (EngHabu) [inline cmd/sandbox/exec.go:20]: Seems to be copied from the `status` command?

File: cmd/sandbox/exec.go [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+func sandboxClusterExec(ctx context.Context, args []string, cmdCtx cmdCore.CommandContext) error {
+	cli, err := docker.GetDockerClient()
+	if err != nil {
+		return err
+	}
+	if len(args) > 0 {
+		return Execute(ctx, cli, args)
+	}
+	return fmt.Errorf("Please use the right syntax. flytectl sandbox exec -- kubectl get pods")
```
Comment (EngHabu) [inline cmd/sandbox/exec.go:30]:
```suggestion
	return fmt.Errorf("Missing argument. Please check usage examples by running flytectl sandbox exec --help")
```

File: go.mod [hunk 3]
```diff
@@ -22,6 +23,7 @@ require (
 	github.com/landoop/tableprinter v0.0.0-20180806200924-8bd8c2576d27
 	github.com/mattn/go-runewidth v0.0.9 // indirect
 	github.com/mitchellh/mapstructure v1.4.1
+	github.com/moby/moby v20.10.7+incompatible // indirect
```
Comment (EngHabu) [inline go.mod:26]: Can you make sure to run `go mod tidy` if you haven't already...

File: pkg/docker/docker_util.go [hunk 4]
```diff
@@ -192,3 +201,26 @@ func GetDockerClient() (Docker, error) {
...
+
+func InspectExecResp(ctx context.Context, cli Docker, containerID string) error {
+	resp, err := cli.ContainerExecAttach(ctx, containerID, types.ExecStartCheck{})
+	if err != nil {
+		return err
+	}
+
+	s := bufio.NewScanner(resp.Reader)
+	for s.Scan() {
+		fmt.Println(s.Text())
```
Comment (EngHabu) [inline pkg/docker/docker_util.go:223]: Should we use fmt.Println or logger.Print ?

Comment (evalsocket) [reply pkg/docker/docker_util.go:223]: no that will  print the output in json and we need plain output

File: cmd/sandbox/exec.go [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+	"fmt"
+
+	cmdCore "github.com/flyteorg/flytectl/cmd/core"
+	"github.com/flyteorg/flytectl/pkg/docker"
+)
+
+const (
+	execShort = "Execute any command in sandbox"
+	execLong  = `
+Execute command will run any command in sandbox.
```
Comment (kumare3) [inline cmd/sandbox/exec.go:14]: Will run non-interactive commands and return immediately with the output?

Comment (evalsocket) [reply cmd/sandbox/exec.go:14]: yes

File: pkg/docker/docker_util.go (hunk 4 above)
Comment (EngHabu) [reply pkg/docker/docker_util.go:223]: ummm... you can configure the logger to output raw text... it'll have the timestamp but will be in raw text not json.
https://github.com/flyteorg/flytestdlib/blob/master/logger/config.go#L17

File: cmd/sandbox/exec.go (hunk 5 above)
Comment (EngHabu) [reply cmd/sandbox/exec.go:14]: I believe Ketan meant to ask you to update the text to say that...

File: pkg/docker/docker_util.go (hunk 4 above)
Comment (EngHabu) [reply pkg/docker/docker_util.go:222]: discussed offline... Yuvraj convinced me to keep it as is

**Review by EngHabu — APPROVED** [review]

**Pull request merged**